modules/Microsoft.ApiManagement/service/deploy.bicep

```diff
@@ -281,7 +294,7 @@ module apiVersionSet_resource 'apiVersionSets/deploy.bicep' = [for (apiVersionSe
   }
 }]
 
-module authorizationServers_resource '.bicep/nested_authorizationServers.bicep' = [for (authorizationServer, index) in authorizationServers: {
+module authorizationServers_resource1 'authorizationServers/deploy.bicep' = [for (authorizationServer, index) in authorizationServerList: {
```

Comment:
Should this be
```suggestion
module authorizationServers_resource 'authorizationServers/deploy.bicep' = [for (authorizationServer, index) in authorizationServerList: {
```

Reply:
No, actually I think it must be this, right?
`server_authorizationServers`

Same for the others....

Reply:
True, I didn't want to open that Pandora's box as I noticed all other resources are also not following the usual naming and that is not related to this PR. I was just wondering if this resource renaming was due to a particular reason.